### PR TITLE
added a RANDOM_SEED as env-level seed for the data faker

### PR DESCRIPTION
**Pull request by Pomax**
Partial fix for https://github.com/mozilla/foundation.mozilla.org/issues/2536

### CODE REVIEW

**Comment by Pomax** [comment]
Nope, I'm down with that.

**Comment by Pomax** [comment]
Added seed value 530910203, which is the CRC16 for "foundation.mozilla.org" =)

File: .travis.yml [hunk 1]
```diff
@@ -19,29 +19,30 @@ script:
...
 - pipenv run flake8 tasks.py network-api/
 - pipenv run coverage run --source './network-api/networkapi' network-api/manage.py test networkapi
-- pipenv run python network-api/manage.py load_fake_data --seed 1
+- pipenv run python network-api/manage.py load_fake_data
 - npm run percy
 after_success:
 - coveralls
 env:
   global:
+  - ALLOWED_HOSTS=localhost
```
Comment (Pomax) [inline .travis.yml]: sorted them alphabetically just because I like being able to find things =D

**Comment by Pomax** [comment]
Hm, it looks like I/we need to figure out how to make the RANDOM_SEED value get ignored during the standard `inv test` part of the run. Because a value is set at the env. level for Travis, `inv test` will use that seed value, and so dates in News won't follow the testing constraints that should apply when no seed is specified =S

**Comment by Pomax** [comment]
Resolved this by making faker instructions that need to be "compliant" during Django's `manage.py test` run but need to be deterministically "random" for visual regressiont esting tap into the `settings.TESTING` var, which tells us whether or not we're in a true test run, or whether we're "just running code".

File: network-api/networkapi/settings.py [hunk 2]
```diff
@@ -538,10 +540,9 @@
 # Buyers Guide Rate Limit Setting
 BUYERS_GUIDE_VOTE_RATE_LIMIT = env('BUYERS_GUIDE_VOTE_RATE_LIMIT')
 
-# Detect if we're testing
+# Detect if we're in official testing mode
```
Comment (patjouk) [inline network-api/networkapi/settings.py]: Nitpicking after PR is merged: can you define official here? Does it mean CI or testing on dev machines?

Comment (Pomax) [reply network-api/networkapi/settings.py]: Neither - the only testing Django knows of is the kind that happens when you run `manage.py test`, which this lets you detect.

**Pull request merged**